Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.csproj

```diff
@@ -13,16 +13,20 @@ Data sources utilized:
 Openflights Airport and Country data available from https://openflights.org/data.html under the Open Database License (https://opendatacommons.org/licenses/odbl/1.0/).
 
 Please only use this component if you agree to and comply with the above license terms.</Description>
-    <Copyright>Copyright © Ibistic Technologies 2018</Copyright>
+    <Copyright>Copyright © Ibistic Technologies 2018-2020</Copyright>
     <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/ibistic/open-airport-data</PackageProjectUrl>
     <RepositoryUrl>https://github.com/ibistic/open-airport-data.git</RepositoryUrl>
     <RepositoryType>Git</RepositoryType>
     <PackageTags>iata, airports, airport</PackageTags>
-    <PackageReleaseNotes>1.0.0: First public release</PackageReleaseNotes>
+    <PackageReleaseNotes>1.1.0:
```

Comment:
Please keep the old release note also. Example: 

<PackageReleaseNotes>
   1.1.0:
      - Accepting duplicates in source data
      - Pinning source data to specific version
      - Upgraded dependency to CsvHelper to version 12.2.1
   1.0.0: First public release
</PackageReleaseNotes>